typings/index.d.ts

```diff
@@ -25,18 +25,34 @@ declare enum InteractionResponseTypes {
   PONG = 1,
   CHANNEL_MESSAGE_WITH_SOURCE = 4,
   DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE = 5,
+  DEFERRED_MESSAGE_UPDATE = 6,
+  UPDATE_MESSAGE = 7,
 }
 
 declare enum InteractionTypes {
   PING = 1,
   APPLICATION_COMMAND = 2,
+  MESSAGE_COMPONENT = 3,
 }
 
 declare enum InviteTargetType {
   STREAM = 1,
   EMBEDDED_APPLICATION = 2,
 }
 
+declare enum MessageButtonStyles {
+  PRIMARY = 1,
+  SECONDARY = 2,
+  SUCCESS = 3,
+  DANGER = 4,
+  LINK = 5,
+}
+
+declare enum MessageComponentTypes {
```

Comment:
This should be added to the `Constants` object.

Reply:
They are, aren't they? (Needs to be changed to MESSAGE_COMPONENT though)
https://github.com/discordjs/discord.js/blob/87465253de0b1da2977039f284af098d390a9d88/src/util/Constants.js#L833

Reply:
I meant in the declarations.